src/main/common/filter.c

```diff
@@ -68,15 +85,26 @@ FAST_CODE float pt1FilterApply(pt1Filter_t *filter, float input)
 
 // PT2 Low Pass filter
 
+// PTn cutoff correction = 1 / sqrt(2^(1/n) - 1)
+#define CUTOFF_CORRECTION_PT2 1.553773974f
+
 FAST_CODE float pt2FilterGain(float f_cut, float dT)
 {
-    // PTn cutoff correction = 1 / sqrt(2^(1/n) - 1)
-    #define CUTOFF_CORRECTION_PT2 1.553773974f
-
     // shift f_cut to satisfy -3dB cutoff condition
     return pt1FilterGain(f_cut * CUTOFF_CORRECTION_PT2, dT);
 }
 
+// Calculates filter gain based on delay (time constant) - time it takes for filter response to reach 63.2% of a step input.
+float pt2FilterGainFromDelay(float delay, float dT)
+{
+    if (delay == 0) {
+        return 1.0f; // gain = 1 means no filtering
+    }
+
+    const float cutoffHz = 1.0f / (M_PIf * delay * CUTOFF_CORRECTION_PT2);
```

Comment:
Now pt1 uses function (pt1FilterCutoffFromDelay), pt2 is inlined. Maybe (I'm not sure) use one approach for both (I'd slightly prefer inlining).

Reply:
i've added `FAST_CODE` to the new functions. So i guess compiler will decide if they are inlined or not. Does it look good now?

Reply:
i was talking about inconsistent style. one version uses helper function, other does not.
generated code is identical, it's just inconsistent.

Reply:
FAST_CODE will place code in (fast) ram and enable optimization for speed. It does not affect inlining directly. 
In this case, it will probably have no effect..

Reply:
ok i think i got it!